packages/data/addon/services/navi-formatter.ts

```diff
@@ -13,7 +13,7 @@ export default class NaviFormatterService extends Service {
     const paramValues = Object.values(allParams);
 
     const name = alias || columnMetadata?.name || '--';
-    if (paramValues.length) {
+    if (paramValues.length && alias === null) {
       return `${name} (${paramValues.join(',')})`;
     } else {
       return name;
```

Comment:
```suggestion
    if (paramValues.length && !alias) {
      return `${name} (${paramValues.join(',')})`;
    } else {
      return alias;
```

Reply:
So I tried this specific format and it caused a few errors with the tests, so I stuck with the original but changes alias === null to !alias